### DISCUSSION TITLE
Login Callbacks Are Not Working

### DISCUSSION THREAD

**Discussion by bartutiryakioglu** [lupidan/apple-signin-unity#147] [Q&A]
Hi, I am having trouble about the login callbacks. I can see that Sign In with Apple ID works as expected but the callbacks are not working so I can't get the required token for credentials? What can I do to resolve this?

**Comment by lupidan** [comment]
**Please** read the documentation to setup the plugin. You are probably not calling `Update` on the manager.
https://github.com/lupidan/apple-signin-unity#initializing
